src/screens/ProfileScreen/index.js

```diff
@@ -195,6 +196,8 @@ const ProfileScreen = ({route}) => {
     reloadFetchAnonymousPost,
     getProfileCache
   } = useProfileScreenHook();
+
+  const {fetchNextFeeds} = useFeedPreloadHook(mainFeeds?.length, () => getMyFeeds(postOffset));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `useFeedPreloadHook` is a good addition for preloading feeds. However, ensure that the hook properly handles edge cases such as when there are no more feeds to fetch or when an error occurs during fetching.



<!-- This is an auto-generated comment by OSS CodeRabbit -->